ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -284,7 +289,16 @@ export type AnalyticsEvent =
     /**
      * Tracks each time the user downloads a cluster's Helm values
      */
-    | typeof LEGACY_CLUSTER_DOWNLOAD_HELM_VALUES;
+    | typeof LEGACY_CLUSTER_DOWNLOAD_HELM_VALUES
+    /**
+     * Tracks each time the user selects a filtered workflow view
+     */
+    | {
+          event: typeof SELECTED_FILTERED_WORKFLOW_VIEW;
+          properties: {
+              value: 'Application View' | 'Platform view' | 'Full view';
```

Comment:
Remind me of the conventions in analytics?
* Any reason for Title Case instead of sentence case and same type?
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/FilteredWorkflowSelector/types.ts#L3-L5
* Taking a step farther out onto a limb, how about the `event` type compared to key in query string?
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/FilteredWorkflowSelector/useFilteredWorkflowURLState.ts#L11

Reply:
The question behind the question is how much can we minimize incremental effort to do something like this?

Reply:
Oops. That was a typo there. It should be in sentence case.

Reply:
For your second point, do you mean how we call the event `Selected Filtered Workflow View` vs. `filteredWorkflowState` in the URL? Do you mean it should be consistent or in the same format? 

I've noticed that our events more or less seem to follow this pattern:

```
[Noun or Subject] [Past Tense Verb]
```
Like `Cluster Created`, `Workload CVE Filter Applied`, and `Vulnerability Report Created`.

So I should follow that pattern here. Instead of `Selected Filtered Workflow View`, it could be `Filtered Workflow View Selected`. I should probably call the URL state `filteredWorkflowView` too. Same for the Component as well. I guess there's a few things we can make more consistent

Reply:
Yeah, that makes sense. Did not expect that we would diverge from the precedent for event keys.

Maybe we can follow up with a comment that has:
* pattern for name that you articulated
* with 3 examples (including this) to guide future contributors, especially to reduce the number of decisions under time pressure